RecoHI/HiEvtPlaneAlgos/python/HiEvtPlane_cfi.py

```diff
@@ -1,25 +1,53 @@
                             vertexTag = cms.InputTag("hiSelectedVertex"),
                             caloTag = cms.InputTag("towerMaker"),
                             castorTag = cms.InputTag("CastorTowerReco"),
                             trackTag = cms.InputTag("hiGeneralTracks"),
-                            centralityBinTag = cms.InputTag("centralityBin","HFtowers"),
-                            centralityVariable = cms.string("HFtowers"),
+                            lostTag = cms.InputTag("lostTracks"),
+                            chi2MapTag = cms.InputTag("packedPFCandidateTrackChi2"),
+                            chi2MapLostTag = cms.InputTag("lostTrackChi2"),
                             nonDefaultGlauberModel = cms.string(""),
-                            FlatOrder = cms.int32(9),
-                            NumFlatBins = cms.int32(40),
-                            CentBinCompression = cms.int32(5),
-                            caloCentRef = cms.double(80.),
-                            caloCentRefWidth = cms.double(5.0),
                             loadDB = cms.bool(False),
                             minet = cms.double(-1.),
-                            maxet = cms.double(-1.),
+                            maxet = cms.double(-1),
                             minpt = cms.double(0.3),
                             maxpt = cms.double(3.0),
                             minvtx = cms.double(-25.),
                             maxvtx = cms.double(25.),
-                            dzerr = cms.double(10.),
-                            chi2 = cms.double(40.)
+                            flatnvtxbins = cms.int32(10),
+                            flatminvtx = cms.double(-15.0),
+                            flatdelvtx = cms.double(3.0),
+                            dzdzerror = cms.double(3.0),
+                            d0d0error = cms.double(3.0),
+                            pterror = cms.double(0.1),
+                            chi2perlayer = cms.double(0.18),
+                            dzdzerror_pix = cms.double(10.),
+                            chi2 = cms.double(40.),
+                            nhitsValid = cms.int32(11),
+                            FlatOrder = cms.int32(9),
+                            NumFlatBins = cms.int32(40),
+                            caloCentRef = cms.double(80.),
+                            caloCentRefWidth = cms.double(5.0),
+                            CentBinCompression = cms.int32(5),
+                            cutEra = cms.int32(2) # 0:ppReco, 1:HIReco, 2:Pixel, 3: GenMC
                             )
+
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(hiEvtPlane,
+    vertexTag = cms.InputTag("offlinePrimaryVertices"),
+    trackTag = cms.InputTag("generalTracks"),
+    minet = cms.double(0.01),
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.
```suggestion
    vertexTag = "offlinePrimaryVertices",
    trackTag = "generalTracks",
    minet = 0.01,
```

similarly for the rest